<?xml version="1.0"?>
<launch>
    <node name="rplidarNode"          pkg="rplidar_ros"  type="rplidarNode" output="screen">
        <param name="serial_port"         type="string" value="/dev/rplidar"/>
        <param name="serial_baudrate"     type="int"    value="115200"/><!--A1/A2 -->
        <!--param name="serial_baudrate"     type="int"    value="256000"--><!--A3 -->
        <param name="frame_id"            type="string" value="laser"/>
        <param name="inverted"            type="bool"   value="false"/>
        <param name="angle_compensate"    type="bool"   value="true"/>
    </node>


    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find rplidar_ros)/rviz/rplidar.rviz" />


    <node type="point_cloud_assembler" pkg="laser_assembler" name="my_assembler">
        <remap from="cloud" to="my_cloud_in"/>
        <param name="max_clouds" type="int" value="400" />
        <param name="fixed_frame" type="string" value="laser" />
    </node>


</launch>



